invenio_app_ils/documents/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -354,7 +354,16 @@
               "type": "keyword"
             },
             "publisher": {
-              "type": "keyword"
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword"
+                },
+                "normalized_keyword": {
```

Comment:
combination of text + keyword + normalised keyword is a bit overkill for this use case, simple text analyser is enough (to be tested)

Reply:
there is still a simpler solution. The normalized keyword analyser was introduced for the specific use case of importer matching, and decreases searching possibilities (`"CErn"` search query would most likely not work) so I would not copy this solution